.github/ISSUE_TEMPLATE/technical-question.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Technical Question
+about: Ask a technical question related to the SDK
+title: ''
+labels: question
+assignees: ''
+
+---
+
+------------------------------- delete below -------------------------------
+
+The best place to submit technical questions related to the SDK is on Stack Overflow. Simply submit a question with the tag "azure-iot-hub". 
+
+Need Technical Support? Every customer with an active Azure subscription has access to support with guaranteed response time. Consider submitting a ticket and get assistance from Microsoft support team using the following link:
```

Comment:
>Need Technical Support? [](start = 0, length = 23)

It's still not clear when I'd want to use the "Technical questions" of line above versus "Need technical support"?  Might want to check with Yi if there's more precise guidanec around this?

If possible it's best if folks can hit up public forums, if only so there's an ongoing record of discussion and other people can search it in future.

Reply:
This is a good point of clarification. Let's let Yi weigh in.

Reply:
I've removed this from the technical-question doc since it isn't relevant.